src/pages/Feeds/Feeds.js

```diff
@@ -45,7 +46,9 @@ class Feeds extends Component {
   };
 
   fetchSoleFeedsData(path) {
-    fetch(path)
+    let isPath =
+      path === '평균별점' ? `${API.addFeeds}${path}` : `${API.feeds}${path}`;
```

Comment:
아래와 같이 작성하시면 코드가 더 직관적일 것 같네요! 

```suggestion
    let isPath = `${API[path === '평균별점' ? 'addFeeds' : 'feeds']}${path}`;
```